src/pixi/display/DisplayObject.js

```diff
@@ -310,130 +317,133 @@ PIXI.DisplayObject.prototype = {
     * @param {PIXI.DisplayObjectContainer} [parent] - Optional parent to calculate this DisplayObjects transform from.
     * @return {PIXI.DisplayObject} - A reference to this DisplayObject.
     */
-    updateTransform: function (parent)
```

Comment:
The updateTransform looks like a giant change, but I believe the only code change is the

if (this.game.config.decomposeMatrix)

code block.

Reply:
If you hit ignore whitespace under diff settings it will have this show up properly